FixedSessionFaker.swift

```diff
@@ -0,0 +1,73 @@
+// Created by Lunar on 15/09/2021.
+//
+
+import Foundation
+import CoreData
+
+class FixedSessionFaker {
+    init(context: NSManagedObjectContext) {
+        let req: NSFetchRequest<SessionEntity> = SessionEntity.fetchRequest()
+        req.predicate = NSPredicate(format: "name LIKE %@", "Its complicated")
+        var session: SessionEntity!
+        if let already = try! context.fetch(req).first {
+            print("A")
+            session = already
+        } else {
+            print("B")
```

Comment:
Have you intentionally left those prints?

Reply:
no, good catch 💯